Packs/Rapid7_Nexpose/Integrations/Rapid7_Nexpose/Rapid7_Nexpose.py

```diff
@@ -148,12 +150,24 @@ def __init__(self, url: str, username: str, password: str, token: str | None = N
 
     def _http_request(self, **kwargs):
         """Wrapper for BaseClient._http_request() that optionally removes `links` keys from responses."""
-        response = super()._http_request(**kwargs)
-
-        if REMOVE_RESPONSE_LINKS:
-            return remove_dict_key(response, "links")
+        for _time in range(1, CONNECTION_ERRORS_RETRIES + 1):
+            try:
+                response = super()._http_request(**kwargs)
+                if REMOVE_RESPONSE_LINKS:
+                    return remove_dict_key(response, "links")
+                return response
+            except (DemistoException, requests.ReadTimeout) as error:
+                demisto.error(f'Error {error} running _http_request in time {_time}')
+                if (
+                    isinstance(error, DemistoException) and not isinstance(
+                        error.exception, requests.ConnectionError
+                    ) or _time == CONNECTION_ERRORS_RETRIES
+                ):
+                    raise
```

Comment:
What is the error raised here?

Reply:
any error that is not connection/read timeout errors